test/helpers/prepare.js

```diff
@@ -0,0 +1,88 @@
+// Native
+const { join } = require('path')
+
+// Packages
+const strip = require('strip-indent')
+const { imageSync: getImageFile } = require('qr-image')
+const { ensureDir, writeFile } = require('fs-extra')
+
+const getDockerFile = session => strip(`
+  FROM mhart/alpine-node:latest
+
+  LABEL name "now-cli-dockerfile-${session}"
+
+  RUN mkdir /app
+  WORKDIR /app
+  COPY package.json /app
+  RUN yarn
+  COPY index.js /app
+
+  EXPOSE 3000
+  CMD ["yarn", "start"]
+`).trim()
```

Comment:
Docker is able to tolerate the indents, and the extra whitespace. Please remove the unnecessary module and extra function call

Reply:
Sweet! Done 😊